scripts/build_site.sh

```diff
@@ -3,10 +3,10 @@ set -e
 
 mkdir -p generated/css generated/js generated/img tmp/js
 
-npx sass src/sass/_all.scss all.css -s compressed --no-source-map
+sass src/sass/_all.scss all.css -s compressed --no-source-map
 mv all.css* generated/css
-npx tsc
+tsc
 
-npx babel-cli --source-maps --minified --no-comments --presets minify tmp/js/constants.js tmp/js/utils.js tmp/js/kbdnav.js tmp/js/themes.js tmp/js/menu.js tmp/js/header.js tmp/js/sidebar.js tmp/js/tabset.js tmp/js/prism.js tmp/js/codeBlocks.js tmp/js/links.js tmp/js/scroll.js tmp/js/overlays.js tmp/js/lang.js tmp/js/callToAction.js --out-file generated/js/all.min.js
-npx babel-cli --source-maps --minified --no-comments --presets minify tmp/js/themes_init.js --out-file generated/js/themes_init.min.js
-npx svgstore -o generated/img/icons.svg src/icons/**/*.svg
+babel --source-maps --minified --no-comments --presets minify tmp/js/constants.js tmp/js/utils.js tmp/js/kbdnav.js tmp/js/themes.js tmp/js/menu.js tmp/js/header.js tmp/js/sidebar.js tmp/js/tabset.js tmp/js/prism.js tmp/js/codeBlocks.js tmp/js/links.js tmp/js/scroll.js tmp/js/overlays.js tmp/js/lang.js tmp/js/callToAction.js --out-file generated/js/all.min.js
```

Comment:
surprised this is in the path?

Reply:
Switching the install location to be --global instead of --save-dev is what makes this work. It installs a .js file in /usr/local/bin for the different tools. Using npx ended up being redundant and was causing all the stuff to be redownloaded upon use. Now, it just works :-)